### DISCUSSION TITLE
ANTLR grammar for Makefile ?

### DISCUSSION THREAD

**Discussion by gidigal** [antlr/antlr4#4519] [Q&A]
Hello,
Is there a grammar for Makefiles ? I did not find one in
https://github.com/antlr/grammars-v4

Thanks,
Gidi

**Comment by ericvergnaud** [comment]
Hi. I guess this issue belongs to grammars-v4 repo ?Envoyé de mon iPhoneLe 26 janv. 2024 à 12:11, gidigal ***@***.***> a écrit :﻿
Hello,
Is there a grammar for Makefiles ? I did not find one in
https://github.com/antlr/grammars-v4
Thanks,
Gidi

**Comment by gidigal** [comment]
Thanks @ericvergnaud , I'll close this discussion and will reopen it in grammars-v4, I was not aware of a separated discussion over there.
